services/audio-broadcaster/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{- range $index, $mic := .Values.microphones }}
+{{ $appName := printf "%s-%s" $.Release.Name $index | trunc 63 | trimSuffix "-" }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $appName }}
+  labels:
+    {{- include "audio-broadcaster.labels" $ | nindent 4 }}
+spec:
+  replicas: {{ $.Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "audio-broadcaster.selectorLabels" $ | nindent 6 }}
+      app.kubernetes.io/instance: {{ $appName }}
+  template:
+    metadata:
+      {{- with $.Values.podAnnotations }}
+      annotations:
+        {{- toYaml $ | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "audio-broadcaster.selectorLabels" $ | nindent 8 }}
+        app.kubernetes.io/instance: {{ $appName }}
+    spec:
+      containers:
+        - name: {{ $appName }}
+          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag }}"
+          imagePullPolicy: {{ $.Values.image.pullPolicy }}
+          env:
+            - name: MICROPHONE_SERVER_HOST
+              value: {{ $mic.host | quote }}
+            - name: MICROPHONE_SERVER_PORT
+              value: {{ $mic.port | quote }}
+          {{- range $env_var, $env_value := $.Values.env }}
+            - name: {{ $env_var }}
+              value: {{ $env_value | quote }}
+          {{- end }}
+          {{- with $.Values.resources }}
+          resources:
+            {{- toYaml $.Values.resources | nindent 12 }}
+          {{- end }}
+      {{- if $.Values.image.nexus3 }}
+      imagePullSecrets:
+        - name: {{ $.Values.namespace }}-{{ $.Values.image.nexus3 }}
+      {{- end }}
+      hostIPC: {{ or $.Values.useHostIpc (eq ( $.Values.useHostIpc | toString ) "<nil>" ) }}
```

Comment:
Theses two lines should be removed. These were only necessary for the DDS systems.